### DISCUSSION TITLE
Track in YOLOv8 | define imgsz param

### DISCUSSION THREAD

**Discussion by Diogo2Analytics** [ultralytics/ultralytics#8812] [Q&A]
hello, im trying to define the imgsz param for model.track. Via the documentation i see that i don't have that option but am i able to define it in the creation of the class perhaps?

What i would live to achieve but know its not possible
`model.track(source=frame,persist=True,imgsz=1312)`

thanks in advance

**Comment by glenn-jocher** [comment]
Hello! 😊 You're correct that the `imgsz` parameter isn't directly available in the `model.track()` method. However, you can set the image size when you load your model, and this size will be used for tracking as well. Here's how you can do it:

```python
from ultralytics import YOLO

# Load the model with the desired imgsz
model = YOLO('yolov8n.pt', imgsz=1312)

# Now, when you use model.track, it will use the imgsz defined earlier
results = model.track(source=frame, persist=True)
```

This way, the `imgsz` you set during model initialization will be applied for tracking. Hope this helps, and happy tracking!
